SU-HKKU/cardinal_service_profile
# CSD-12 Opportunities filters built with React and PDB

# READY FOR REVIEW

# Summary
- Search API view for opportunities results
- Progressively decoupled blocks module to build a custom filter UI for the user
- add some more fields to the opportunity content type with taxonomy vocabs

# Need Review By (Date)
- 4/29

# Urgency
- medium

# Steps to Test
1. I set up a page on tugboat for quick verification: https://pr8-duzhkv3nisyijofdbmuqrmfsemrk7qjz.tugboat.qa/search-opportunities
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-12-filters-react`
1. `blt drupal:install --site=cardinalservice` or `blt drupal:update --site=cardinalservice` if you already have a site.
1. create some opportunity content via devel, manually, or `drush @cardinalservice.local mim solo_opportunities`
1. create an empty page
1. edit the layout and add the "Opportunities Filtering List" block & the "Opportunities: All Filtered" blocks
1. after saving the layout, verify the filters function and you see some results (styling to come later)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
